contracts/FlowCallbackScheduler.cdc

```diff
@@ -26,58 +26,70 @@ access(all) contract FlowCallbackScheduler {
     }
 
     /// Events
-    access(all) event CallbackScheduled(
+    access(all) event Scheduled(
         id: UInt64,
-        timestamp: UFix64?,
         priority: UInt8,
+        timestamp: UFix64?,
         executionEffort: UInt64,
         fees: UFix64,
         callbackOwner: Address
     )
 
-    access(all) event CallbackProcessed(
+    access(all) event Processed(
         id: UInt64,
         priority: UInt8,
         executionEffort: UInt64,
         callbackOwner: Address
     )
 
-    access(all) event CallbackExecuted(
+    access(all) event Executed(
         id: UInt64,
         priority: UInt8,
+        executionEffort: UInt64,
+        fees: UFix64,
         callbackOwner: Address,
-        fees: UFix64
+        succeeded: Bool
```

Comment:
Cool idea, but I think if we implement failures for callbacks, we will likely also add a status for callback which will be succeeded / failed and executed status will be removed. So I believe we will have two events succeeded and failed. So not sure about adding this field.

Reply:
What do you mean by "if we implement failures", don't we have to implement failures? I assumed that is was mandatory that we defined a standard way to handle failures.